### PR TITLE
Release Note v11.3.0

### PR DESCRIPTION
**Pull request by albertotessarotto**
## Pull Request Type

- [ ] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [ ] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [ ] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [ ] No sensible content has been committed

### CODE REVIEW

File: docs/release-notes/v11.3.0.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+- Values of properties with the same name in different components are no longer bound
+- Fixed references resolution in the preview
+- When the compose application is removed, its metadata are deleted
+
+## Fast Data
+
+### New Features
+
+#### Fast Data No Code Configuration for Aggregation
```
Comment (TheRusCompass) [inline docs/release-notes/v11.3.0.md:71]: @ThisIsDemetrio @fedepini can we add a screenshot here?

Comment (fedepini) [reply docs/release-notes/v11.3.0.md:71]: @TheRusCompass done!

File: docs/release-notes/v11.3.0.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+- CRUDs tab becomes **Data Models** and includes also Single Views and Projections collections
+- **contains** operator to allow case insensitive free search on text fields
+- **not set** operator to filter null values
+- fix graphic issues
+
+For more information, [go to the documentation](/development_suite/governance/design-overview.md).
+
+![Design Overview](./img/overview/design-overview-update.png)
+
+#### `⌘ + S` (`ctrl + S`) shortcut for saving configuration
```
Comment (albertotessarotto) [inline docs/release-notes/v11.3.0.md:42]:
```suggestion
#### Shortcut for saving configuration process:`⌘ + S` (`ctrl + S`)
```

File: docs/release-notes/v11.3.0.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+
+It is possible to use `⌘ + K` (alternatively, `ctrl + K`) to speed-up your saving process, allowing to automatically open the Save configuration modal.
```
Comment (albertotessarotto) [inline docs/release-notes/v11.3.0.md:44]:
```suggestion
It is possible to use `⌘ + S` (alternatively, `ctrl + S`) to speed-up your saving process, allowing to automatically open the Save configuration modal.
```

File: docs/release-notes/v11.3.0.md [hunk 4]
```diff
@@ -39,9 +39,9 @@ For more information, [go to the documentation](/development_suite/governance/de
 
 ![Design Overview](./img/overview/design-overview-update.png)
 
-#### `⌘ + S` (`ctrl + S`) shortcut for saving configuration
+#### Shortcut for saving configuration process:`⌘ + S` (`ctrl + S`)
 
-It is possible to use `⌘ + K` (alternatively, `ctrl + K`) to speed-up your saving process, allowing to automatically open the Save configuration modal.
+It is possible to use `⌘ + S` (alternatively, `ctrl + S`) to speed-up your saving process, allowing to automatically open the Save configuration modal.
```
Comment (fredmaggiowski) [inline docs/release-notes/v11.3.0.md:44]: let's put here a notice about being forced to save on another branch if someone has already saved on the same branch

File: docs/release-notes/v11.3.0.md [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+![Webhooks Section](./img/overview/webhooks_section.PNG)
+
+Starting from today, it will be possible to view, create, edit and delete custom Console Webhooks directly through the user interface.
+Whereas before today this was only possible through Console CMS, now the user can enter the "Company Overview" area and under the "Webhooks" section can easily and intuitively manage all the respective hooks.
+
+For more information, [go to the documentation]().
+
+#### Configurable `defaultAnnotations` and `defaultLabels` for microservices
+
+From now on, within the CMS you can configure `defaultAnnotations` and `defaultLabels` properties for your Marketplace microservices. For more information about microservice properties, visit the [related page](../marketplace/add_to_marketplace/contributing_overview#common-to-microservice-components-plugins-templates-examples)
```
Comment (albertotessarotto) [inline docs/release-notes/v11.3.0.md:25]:
```suggestion
From now on, within the CMS you can configure `defaultAnnotations` and `defaultLabels` properties for your Marketplace microservices. For more information about microservice properties, visit the [related page](../marketplace/add_to_marketplace/contributing_overview#common-to-microservice-components-plugins-templates-examples).
```

File: docs/release-notes/v11.3.0.md (hunk 4 above)
Comment (albertotessarotto) [reply docs/release-notes/v11.3.0.md:44]: inserted a note about

File: docs/release-notes/v11.3.0.md [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+:::
+
+#### Addition of ignoreTrailingSlash flag in manual routes configuration
+
+In Authorization section, when configuring a [manual route](/development_suite/api-console/api-design/authorization#manual-routes-tab), it is now possible to set the `ignoreTrailingSlash` flag. When enabled, a path registered with a trailing slash is considered the same path as the one without it, and viceversa.
+To know more about manual routes configuration options, visit the official [Rönd documentation](https://rond-authz.io/docs/configuration#route-specific-options).
+
+### Bug Fix
+
+This version addressed a few bugs, here is a comprehensive list:
```
Comment (marcofilippi) [inline docs/release-notes/v11.3.0.md:57]: If there's just one bug fix, I'd rephrase it

File: docs/release-notes/v11.3.0.md [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+
+##### `bk-button` is temporarily disabled on file download/upload
+
+`disbaleOnAction` property of `bk-button` correctly interacts with `bk-file-client`.
+
+## Marketplace
+
+### Marketplace Updates
+
+#### CRUD Service
```
Comment (albertotessarotto) [inline docs/release-notes/v11.3.0.md:154]:
```suggestion
#### CRUD Service - v6.6.1
```

File: docs/release-notes/v11.3.0.md [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+_June 22th, 2023_
+
+## Console
+
+### New Features
+
+#### Manage and configure Webhooks directly via Console UI
+
+Starting from today, it will be possible to view, create, edit and delete custom Console Webhooks directly through the user interface.
+Whereas before today this was only possible through Console CMS, now the user can enter the "Company Overview" area and under the "Webhooks" section can easily and intuitively manage all the respective hooks.
```
Comment (marcofilippi) [inline docs/release-notes/v11.3.0.md:17]:
```suggestion
#### Manage and configure Webhooks directly via Console UI

Starting from today, it will be possible to view, create, edit and delete custom Console Webhooks directly through the user interface.
Whereas before today this was only possible through Console CMS, now the user can enter the Company Overview area and easily manage all the respective hooks under the new "Webhooks" section.
```

**Review by pmarchini — APPROVED** [review]
